yarn-project/ethereum/src/test/tx_delayer.ts

```diff
@@ -190,7 +190,7 @@ export function withDelayer<T extends ViemClient>(
             maxInclusionTimeIntoSlot: delayer.maxInclusionTimeIntoSlot,
           };
 
-          if (now - Number(lastBlockTimestamp) > delayer.maxInclusionTimeIntoSlot) {
+          if (now / 1000 - Number(lastBlockTimestamp) > delayer.maxInclusionTimeIntoSlot) {
```

Comment:
Oh I see `dateProvider.nowInSeconds()` truncates the milliseconds so it the delayer thinks it's actually in the past